### DISCUSSION TITLE
Avoir le nb de visiteurs uniques dans Matomo

### DISCUSSION THREAD

**Discussion by rtaieb** [betagouv/cap#20] [Besoin]
### Besoin

```bash
Dans Matomo, on devrait pouvoir tracer le nombre de visiteurs uniques sur notre site.

Cela nous permettrait d'analyser plus finement le parcours de nos utilisateurs, notamment de ceux qui vont valider un tunnel de conversion en plusieurs visites.
Cela nous permettrait également d'avoir une vision plus précise du nb de personnes utilisant notre site.

Pour information cette métrique était disponible dans l'instance Matomo stats.data.gouv.fr que nous utilisions avant la migration de décembre 2023.
```

### Solutions envisageables

```bash
Actuellement, il n'est pas possible de suivre le nb de visiteurs unique en utilisant le tracking via le Matomo de Beta car l'option "Forcer le suivi sans cookies" est cochée.

Une solution serait donc de décocher cette option, et de laisser les startup gérer le consentement cookie de ses utilisateurs.
```

### Informations complémentaires

- [ ] J'avais déjà entendu ce besoin au sein de la communauté beta.gouv
- [X] Je me porte volontaire pour participer à la mise en place de la future solution choisie

**Comment by jdauphant** [comment]
Pourquoi le paramétre "Forcer le suivi sans cookies" a été activé ? ça ne fait pas partie des recommandations cnil pour être en mode exempté : https://www.cnil.fr/sites/cnil/files/atoms/files/matomo_analytics_-_exemption_-_guide_de_configuration.pdf

**Reply by adipasquale** [reply]
super ce doc, merci @jdauphant

**Comment by fcoufour** [comment]
@rtaieb Raphaël, quand tu dis "il n'est pas possible", tu veux bien dire que les données de visiteur unique mensuel sont présentes mais qu'elles ne sont pas justes ? ou tu veux dire qu'il n'y pas du tout de données de visiteur unique sur une période de temps libre ?

**Reply by rtaieb** [reply]
@fcoufour 
Je veux dire qu'il n'est pas possible de distinguer le nb de visiteurs et le nb de visiteurs uniques.
Ces deux métriques sont identiques dans Matomo, et ont la valeur du nb de visiteurs.

**Reply by fcoufour** [reply]
Il n'y a pas de "nombre de visiteurs" sur Matomo, seulement un nombre de visites et un nombre de visiteurs uniques. Et le nombre de visiteurs uniques est plus ou moins juste selon la période et la configuration de Matomo (sans cookies comme actuellement ou avec un cookie avec une certaine durée d'expiration). Tu constates que ton nombre de visiteurs uniques est égal à ton nombre de visites sur la période d'intégration à cette instance Matomo ?

**Reply by rtaieb** [reply]
Oui tu as raison, je voulais dire que nous avons, comme beaucoup d'autres startup suivies sur stats.beta.gouv.fr un nombre de visiteurs uniques égal au nombre de visites.

Par exemple pour notre SE : https://stats.beta.gouv.fr/index.php?module=CoreHome&action=index&idSite=81&period=day&date=yesterday#?period=month&date=2024-05-01&idSite=81&category=General_Visitors&subcategory=General_Overview

Ou pour Impact CO2 : https://stats.beta.gouv.fr/index.php?module=CoreHome&action=index&idSite=90&period=day&date=yesterday#?period=month&date=2024-05-01&idSite=90&category=General_Visitors&subcategory=General_Overview

Ou le taxi : https://stats.beta.gouv.fr/index.php?module=CoreHome&action=index&idSite=4&period=day&date=yesterday#?period=month&date=2024-05-01&idSite=4&category=General_Visitors&subcategory=General_Overview


Pour nous, ces deux nombres étaient différents et semblaient plus refléter la réalité lorsque nous étions sur stats.data.gouv.fr

**Reply by fcoufour** [reply]
Très clair donc quasi même constat pour ma SE ROMIN mais pas depuis le début, depuis fin janvier (nous sommes sur l'instance depuis le début du produit - mars 2023)

**Comment by revolunet** [comment]
Donc il s'agit de ce paramètre <img width="1303" alt="Capture d’écran 2024-06-10 à 11 34 36" src="https://github.com/betagouv/cap/assets/124937/e68fae5f-d88f-4790-9cf7-8b901059fb88">

**Comment by revolunet** [comment]
Le paramètre "Forcer le suivi sans cookies" a été désactivé 🎉
